src/component/TableBody.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from "react";
+import moment from "moment";
+
+const TableRow = props => {
+  const [highLight, setHighlight] = useState("noColor");
+  const highlightRow = () => {
+    setHighlight(highLight => {
+      if (highLight === "noColor") {
+        return "color";
+      } else {
+        return "noColor";
+      }
+    });
+  };
+
+  const handleShowProfile = () => {
+    props.selectCustomer(props.booking.id);
```

Comment:
If you add event as an arguement and 
`event.stopPropagation();`

It would stop the row highlighting when you click the button